packages/react-textarea/Spec.md

```diff
@@ -66,11 +65,11 @@ In some libraries, `TextArea` is a variant of `TextField` or `Input`.
 ```ts
 type TextAreaProps = {
   /**
-   * Display inline instead of block.
+   * How shall the TextArea be displayed.
    *
-   * @defaultvalue false
+   * @defaultvalue 'inline-block'
    */
-  inline?: boolean;
+  display?: 'inline-block' | 'inline-flex' | 'inline-grid';
```

Comment:
I'm realizing I worded my last comment on this poorly. This prop should be removed entirely to minimize our component APIs, imo. 

The default UA display style for all input controls is `inline-block` so when we apply styles to our components we should align with this and use `inline-block/flex/grid` but not expose a prop to control this on the component.

Ideally users should be able to control the layout of controls like this via a parent container with flexbox or CSS grid. They can also override our styles if necessary.